schemas/bedrock-profile-http.js

```diff
@@ -0,0 +1,148 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const account = {
+  title: 'Account',
+  type: 'string'
+};
+
+const profile = {
+  title: 'Profile',
+  type: 'string'
+};
+
+// this should match query objects with an account in them
+const accountQuery = {
+  title: 'Accout Query',
+  type: 'object',
+  required: ['account'],
+  // FIXME set this to false once we have
+  // decided on all the properties.
+  additionalProperties: true,
+  properties: {account}
+};
+
+const profileAgent = {
+  title: 'Profile Agent',
+  type: 'object',
+  required: ['profile'],
+  // FIXME set this to false once we have
+  // decided on all the properties.
+  additionalProperties: true,
+  properties: {
+    account,
+    profile,
+    token: {
+      title: 'Token',
+      type: 'string'
+    }
+  }
+};
+
+const profileAgents = {
+  title: 'Profile Agents',
+  type: 'object',
+  // FIXME set this to false once we have
+  // decided on all the properties.
+  additionalProperties: true,
+  properties: {
+    account,
+    profile
+  }
+};
+
+const delegateCapability = {
+  title: 'Delegate Capability',
+  type: 'object',
+  required: ['invoker', 'account'],
+  // FIXME set this to false if we
+  // have decided on all the properties
+  additionalProperties: true,
+  properties: {
+    account,
+    invoker: {
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'array',
+        minItems: 1,
+        items: [{type: 'string'}]
+      }]
+    }
+  }
+};
+
+const zcap = {
+  title: 'zcap',
+  type: 'object',
+  // FIXME set this to false once we have
+  // decided on all the properties.
+  additionalProperties: true,
+  properties: {
+    id: {
+      title: 'id',
+      type: 'string'
+    },
+    controller: {
+      title: 'controller',
+      type: 'string'
+    },
+    allowedAction: {
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'array',
+        minItems: 1,
+        items: [{type: 'string'}]
+      }]
+    },
+    invoker: {
```

Comment:
zcaps can also have `delegator`; it looks like `invoker` does.

Reply:
I added `delegator` to the zcap. It is also missing `referenceId`, `parentCapability`, and `proof`. https://github.com/digitalbazaar/bedrock-profile-http/pull/10/commits/db5cc0508923ab42a904fe0de5931d4090682b52